setup.py

```diff
@@ -0,0 +1,48 @@
+from setuptools import setup, find_packages
+
+setup(
+  name = 'autocards',
+  packages = find_packages(exclude=[]),
+  include_package_data = True,
+  entry_points={
+    'console_scripts': [
+      'autocard = autocards.cli:main'
+    ],
+  },
+  version = '0.0.1',
+  license='MIT',
+  description = 'Accelerating learning through machine-generated flashcards.',
+  author = 'Paul Bricman',
+  author_email = 'paulbricman@protonmail.com',
+  long_description_content_type = 'text/markdown',
+  url = 'https://github.com/ncoop57/Autocards/tree/master',
+  keywords = [
+    'artificial intelligence',
+    'deep learning',
+    'text to image'
+  ],
+  install_requires=[
+    'PyPDF2 == 1.26.0',
+    'beautifulsoup4 == 4.9.3',
+    'fastcore == 1.4.2',
+    'huggingface_hub == 0.6.0',
```

Comment:
For clarification, your commit's goal is to make it into a lib but will it be also pushed to hugging face? And have you considered adding it to PyPI ?

Reply:
The huggingface hub thing was not supposed to be there, I was playing around with some dependencies and forgot to remove that. As for uploading it pypi, I can do that, but I wasn't sure if @paulbricman you were interested in having the keys to the kingdom so to speak for managing the pypi package. If not, I can get it setup on PyPi

Reply:
In the meantime, can you double check the dependencies to make sure all are needed ? That means removing huggingface_hub to start with.

For the PyPi let's wait on @paulbricman's answer